node/src/Context.cpp

```diff
@@ -71,7 +70,8 @@ namespace node {
 
         WebnnProcTable backendProcs = webnn_native::GetProcs();
         webnnProcSetProcs(&backendProcs);
-        mImpl = ml::Context::Acquire(webnn_native::CreateContext(&options));
+        mImpl = ml::Context::Acquire(
+            webnn_native::CreateContext(reinterpret_cast<MLContextOptions const*>(&options)));
```

Comment:
It's C API, so it's better to keep current implemenation and set the default value to MLContextOptions like
`MLContextOptions options = {MLPowerPreference_Default,  MLDevicePreference_Default};`

Reply:
OK, thanks!